compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangTypeInit.java

```diff
@@ -36,7 +36,7 @@
 
     public BLangType userDefinedType;
     public List<BLangExpression> argsExpr;
-    public BLangInvocation initInvocation;
+    public BLangExpression initInvocation;
```

Comment:
Why is this being changed?

Reply:
I have changed 'BLangInvocation' into a `StatementExpression`. As an example 
```ballerina
function value(int a, int b = 10 + a) {

}

public function main() {
    int $a$ = 10;
    int $b$ = value_b(a);
    value($a$, $b$)
}
```